2021-05-09 11:14:28,414 INFO    MainThread:12924 [wandb_setup.py:_flush():69] setting env: {}
2021-05-09 11:14:28,415 INFO    MainThread:12924 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-09 11:14:28,415 INFO    MainThread:12924 [wandb_init.py:_log_setup():336] Logging user logs to c:\Users\Steven\Documents\Github\ML\Proyectos\Proyecto II\wandb\run-20210509_111428-10smt1az\logs\debug.log
2021-05-09 11:14:28,416 INFO    MainThread:12924 [wandb_init.py:_log_setup():337] Logging internal logs to c:\Users\Steven\Documents\Github\ML\Proyectos\Proyecto II\wandb\run-20210509_111428-10smt1az\logs\debug-internal.log
2021-05-09 11:14:28,417 INFO    MainThread:12924 [wandb_init.py:init():369] calling init triggers
2021-05-09 11:14:28,417 INFO    MainThread:12924 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {'epochs': 600, 'learningRate': 0.0001, 'batchSize': 128, 'inputSize': 256, 'fc_size_1': 256, 'fc_size_2': 128, 'fc_size_3': 64, 'fc_size_4': 32, 'fc_size_5': 16, 'fc_size_6': 8, 'outputSize': 4, 'classes': ['COVID', 'Lung Opacity', 'Normal', 'Viral Pneumonia'], 'classesLen': 4, 'upSampling': False}
2021-05-09 11:14:28,418 INFO    MainThread:12924 [wandb_init.py:init():389] re-initializing run, found existing run on stack: 60loqspo
2021-05-09 11:14:28,420 INFO    MainThread:12924 [wandb_run.py:finish():1208] finishing run tecai/MLP/60loqspo
2021-05-09 11:14:28,423 ERROR   MainThread:12924 [jupyter.py:save_history():312] Run pip install nbformat to save notebook history
2021-05-09 11:14:28,423 INFO    MainThread:12924 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-09 11:14:28,423 INFO    MainThread:12924 [wandb_init.py:_jupyter_teardown():268] cleaning up jupyter logic
2021-05-09 11:14:28,423 INFO    MainThread:12924 [wandb_run.py:_atexit_cleanup():1535] got exitcode: 0
2021-05-09 11:14:28,424 INFO    MainThread:12924 [wandb_run.py:_restore():1507] restore
2021-05-09 11:14:28,443 INFO    MainThread:12924 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 3839
  total_bytes: 3839
}

2021-05-09 11:14:33,560 INFO    MainThread:12924 [wandb_run.py:_wait_for_finish():1657] got exit ret: None
2021-05-09 11:14:38,684 INFO    MainThread:12924 [wandb_run.py:_wait_for_finish():1657] got exit ret: None
2021-05-09 11:14:43,796 INFO    MainThread:12924 [wandb_run.py:_wait_for_finish():1657] got exit ret: None
2021-05-09 11:14:48,906 INFO    MainThread:12924 [wandb_run.py:_wait_for_finish():1657] got exit ret: None
2021-05-09 11:14:54,021 INFO    MainThread:12924 [wandb_run.py:_wait_for_finish():1657] got exit ret: None
2021-05-09 11:14:59,139 INFO    MainThread:12924 [wandb_init.py:init():418] starting backend
2021-05-09 11:14:59,139 INFO    MainThread:12924 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=spawn, using: spawn
2021-05-09 11:14:59,145 INFO    MainThread:12924 [backend.py:ensure_launched():132] starting backend process...
2021-05-09 11:14:59,249 INFO    MainThread:12924 [backend.py:ensure_launched():136] started backend process with pid: 5724
2021-05-09 11:14:59,250 INFO    MainThread:12924 [wandb_init.py:init():423] backend started and connected
2021-05-09 11:14:59,253 INFO    MainThread:12924 [wandb_init.py:init():465] updated telemetry
2021-05-09 11:14:59,254 INFO    MainThread:12924 [wandb_init.py:init():484] communicating current version
2021-05-09 11:15:00,692 WARNING MainThread:12924 [wandb_init.py:init():764] interrupted
Traceback (most recent call last):
  File "C:\Python39\lib\site-packages\wandb\sdk\wandb_init.py", line 744, in init
    run = wi.init()
  File "C:\Python39\lib\site-packages\wandb\sdk\wandb_init.py", line 485, in init
    ret = backend.interface.communicate_check_version(
  File "C:\Python39\lib\site-packages\wandb\sdk\interface\interface.py", line 789, in communicate_check_version
    result = self._communicate(rec)
  File "C:\Python39\lib\site-packages\wandb\sdk\interface\interface.py", line 537, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "C:\Python39\lib\site-packages\wandb\sdk\interface\interface.py", line 82, in get
    is_set = self._object_ready.wait(timeout)
  File "C:\Python39\lib\threading.py", line 574, in wait
    signaled = self._cond.wait(timeout)
  File "C:\Python39\lib\threading.py", line 316, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
2021-05-09 11:15:05,333 INFO    MainThread:12924 [wandb_settings.py:_apply_source_login():713] setting login settings: {}
2021-05-09 11:15:05,348 INFO    MainThread:12924 [wandb_setup.py:_flush():69] setting env: {}
2021-05-09 11:15:05,348 INFO    MainThread:12924 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-09 11:15:05,349 INFO    MainThread:12924 [wandb_init.py:_log_setup():336] Logging user logs to c:\Users\Steven\Documents\Github\ML\Proyectos\Proyecto II\wandb\run-20210509_111505-1oqem14f\logs\debug.log
2021-05-09 11:15:05,349 INFO    MainThread:12924 [wandb_init.py:_log_setup():337] Logging internal logs to c:\Users\Steven\Documents\Github\ML\Proyectos\Proyecto II\wandb\run-20210509_111505-1oqem14f\logs\debug-internal.log
2021-05-09 11:15:05,350 INFO    MainThread:12924 [wandb_init.py:_jupyter_setup():287] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x0000022AF498C850>
2021-05-09 11:15:05,350 INFO    MainThread:12924 [wandb_init.py:init():369] calling init triggers
2021-05-09 11:15:05,351 INFO    MainThread:12924 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {'epochs': 600, 'learningRate': 0.0001, 'batchSize': 128, 'inputSize': 256, 'fc_size_1': 256, 'fc_size_2': 128, 'fc_size_3': 64, 'fc_size_4': 32, 'fc_size_5': 16, 'fc_size_6': 8, 'outputSize': 4, 'classes': ['COVID', 'Lung Opacity', 'Normal', 'Viral Pneumonia'], 'classesLen': 4, 'upSampling': False}
2021-05-09 11:15:05,351 INFO    MainThread:12924 [wandb_init.py:init():418] starting backend
2021-05-09 11:15:05,351 INFO    MainThread:12924 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=spawn, using: spawn
2021-05-09 11:15:05,355 INFO    MainThread:12924 [backend.py:ensure_launched():132] starting backend process...
2021-05-09 11:15:05,458 INFO    MainThread:12924 [backend.py:ensure_launched():136] started backend process with pid: 14048
2021-05-09 11:15:05,459 INFO    MainThread:12924 [wandb_init.py:init():423] backend started and connected
2021-05-09 11:15:05,462 INFO    MainThread:12924 [wandb_init.py:init():465] updated telemetry
2021-05-09 11:15:05,464 INFO    MainThread:12924 [wandb_init.py:init():484] communicating current version
2021-05-09 11:15:07,235 INFO    MainThread:12924 [wandb_init.py:init():489] got version response 
2021-05-09 11:15:07,235 INFO    MainThread:12924 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-09 11:15:07,700 INFO    MainThread:12924 [wandb_init.py:init():522] starting run threads in backend
2021-05-09 11:15:08,860 INFO    MainThread:12924 [wandb_run.py:_console_start():1565] atexit reg
2021-05-09 11:15:08,861 INFO    MainThread:12924 [wandb_run.py:_redirect():1412] redirect: SettingsConsole.WRAP
2021-05-09 11:15:08,862 INFO    MainThread:12924 [wandb_run.py:_redirect():1455] Wrapping output streams.
2021-05-09 11:15:08,862 INFO    MainThread:12924 [wandb_run.py:_redirect():1479] Redirects installed.
2021-05-09 11:15:08,863 INFO    MainThread:12924 [wandb_init.py:init():546] run started, returning control to user process
2021-05-09 11:15:08,863 INFO    MainThread:12924 [wandb_watch.py:watch():39] Watching
2021-05-09 11:27:02,690 INFO    MainThread:12924 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-09 11:27:02,690 INFO    MainThread:12924 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-09 11:27:02,896 INFO    MainThread:12924 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-09 11:27:02,899 INFO    MainThread:12924 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-09 11:27:02,900 INFO    MainThread:12924 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-09 11:29:03,036 INFO    MainThread:12924 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-09 11:29:03,038 INFO    MainThread:12924 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-09 11:29:03,039 INFO    MainThread:12924 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-09 11:29:03,064 INFO    MainThread:12924 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-09 11:29:03,065 INFO    MainThread:12924 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-09 11:29:03,066 INFO    MainThread:12924 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-09 11:29:03,091 INFO    MainThread:12924 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-09 11:29:03,092 INFO    MainThread:12924 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-09 11:29:03,093 INFO    MainThread:12924 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-09 11:29:03,124 INFO    MainThread:12924 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-09 11:29:03,126 INFO    MainThread:12924 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-09 11:29:03,127 INFO    MainThread:12924 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-09 11:29:03,153 INFO    MainThread:12924 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-09 11:29:03,157 INFO    MainThread:12924 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-09 11:29:03,158 INFO    MainThread:12924 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-09 11:29:03,186 INFO    MainThread:12924 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-09 11:29:03,187 INFO    MainThread:12924 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-09 11:29:03,188 INFO    MainThread:12924 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-09 11:29:03,218 INFO    MainThread:12924 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-09 11:29:03,220 INFO    MainThread:12924 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-09 11:29:03,221 INFO    MainThread:12924 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-09 11:29:03,247 INFO    MainThread:12924 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-09 11:29:03,248 INFO    MainThread:12924 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-09 11:29:03,249 INFO    MainThread:12924 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-09 11:29:03,281 INFO    MainThread:12924 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-09 11:29:03,282 INFO    MainThread:12924 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-09 11:29:03,283 INFO    MainThread:12924 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-09 11:29:03,313 INFO    MainThread:12924 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-09 11:29:04,011 INFO    MainThread:12924 [wandb_setup.py:_flush():69] setting env: {}
2021-05-09 11:29:04,011 INFO    MainThread:12924 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-09 11:29:04,012 INFO    MainThread:12924 [wandb_init.py:_log_setup():336] Logging user logs to c:\Users\Steven\Documents\Github\ML\Proyectos\Proyecto II\wandb\run-20210509_112904-bb66joc7\logs\debug.log
2021-05-09 11:29:04,013 INFO    MainThread:12924 [wandb_init.py:_log_setup():337] Logging internal logs to c:\Users\Steven\Documents\Github\ML\Proyectos\Proyecto II\wandb\run-20210509_112904-bb66joc7\logs\debug-internal.log
2021-05-09 11:29:04,013 INFO    MainThread:12924 [wandb_init.py:init():369] calling init triggers
2021-05-09 11:29:04,014 INFO    MainThread:12924 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {'epochs': 600, 'learningRate': 0.0001, 'batchSize': 128, 'inputSize': 256, 'fc_size_1': 256, 'fc_size_2': 128, 'fc_size_3': 64, 'fc_size_4': 32, 'fc_size_5': 16, 'fc_size_6': 8, 'outputSize': 4, 'classes': ['COVID', 'Lung Opacity', 'Normal', 'Viral Pneumonia'], 'classesLen': 4, 'upSampling': True}
2021-05-09 11:29:04,014 INFO    MainThread:12924 [wandb_init.py:init():389] re-initializing run, found existing run on stack: 1oqem14f
2021-05-09 11:29:04,016 INFO    MainThread:12924 [wandb_run.py:finish():1208] finishing run tecai/MLP/1oqem14f
2021-05-09 11:29:04,019 ERROR   MainThread:12924 [jupyter.py:save_history():312] Run pip install nbformat to save notebook history
2021-05-09 11:29:04,019 INFO    MainThread:12924 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-09 11:29:04,020 INFO    MainThread:12924 [wandb_init.py:_jupyter_teardown():268] cleaning up jupyter logic
2021-05-09 11:29:04,020 INFO    MainThread:12924 [wandb_run.py:_atexit_cleanup():1535] got exitcode: 0
2021-05-09 11:29:04,021 INFO    MainThread:12924 [wandb_run.py:_restore():1507] restore
2021-05-09 11:29:04,395 INFO    MainThread:12924 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 3839
  total_bytes: 3839
}

2021-05-09 11:29:04,518 INFO    MainThread:12924 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 3839
  total_bytes: 3839
}

2021-05-09 11:29:04,838 INFO    MainThread:12924 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 3839
  total_bytes: 3839
}

2021-05-09 11:29:05,570 INFO    MainThread:12924 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 33529
  total_bytes: 33529
}

2021-05-09 11:29:05,678 INFO    MainThread:12924 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 33529
  total_bytes: 33529
}

2021-05-09 11:29:05,788 INFO    MainThread:12924 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 33529
  total_bytes: 33529
}

2021-05-09 11:29:05,896 INFO    MainThread:12924 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 33529
  total_bytes: 33529
}

2021-05-09 11:29:06,005 INFO    MainThread:12924 [wandb_run.py:_wait_for_finish():1657] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 33529
  total_bytes: 33529
}

2021-05-09 11:29:07,251 INFO    MainThread:12924 [wandb_run.py:_show_summary():1812] rendering summary
2021-05-09 11:29:07,258 INFO    MainThread:12924 [wandb_run.py:_show_history():1850] rendering history
2021-05-09 11:29:07,265 INFO    MainThread:12924 [wandb_run.py:_show_files():1879] logging synced files
2021-05-09 11:29:07,278 INFO    MainThread:12924 [wandb_init.py:init():418] starting backend
2021-05-09 11:29:07,279 INFO    MainThread:12924 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=spawn, using: spawn
2021-05-09 11:29:07,287 INFO    MainThread:12924 [backend.py:ensure_launched():132] starting backend process...
2021-05-09 11:29:07,471 INFO    MainThread:12924 [backend.py:ensure_launched():136] started backend process with pid: 6760
2021-05-09 11:29:07,472 INFO    MainThread:12924 [wandb_init.py:init():423] backend started and connected
2021-05-09 11:29:07,476 INFO    MainThread:12924 [wandb_init.py:init():465] updated telemetry
2021-05-09 11:29:07,477 INFO    MainThread:12924 [wandb_init.py:init():484] communicating current version
2021-05-09 11:29:08,955 INFO    MainThread:12924 [wandb_init.py:init():489] got version response 
2021-05-09 11:29:08,955 INFO    MainThread:12924 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-09 11:29:09,369 INFO    MainThread:12924 [wandb_init.py:init():522] starting run threads in backend
2021-05-09 11:29:10,475 INFO    MainThread:12924 [wandb_run.py:_console_start():1565] atexit reg
2021-05-09 11:29:10,477 INFO    MainThread:12924 [wandb_run.py:_redirect():1412] redirect: SettingsConsole.WRAP
2021-05-09 11:29:10,477 INFO    MainThread:12924 [wandb_run.py:_redirect():1455] Wrapping output streams.
2021-05-09 11:29:10,478 INFO    MainThread:12924 [wandb_run.py:_redirect():1479] Redirects installed.
2021-05-09 11:29:10,478 INFO    MainThread:12924 [wandb_init.py:init():546] run started, returning control to user process
2021-05-09 11:29:10,478 INFO    MainThread:12924 [wandb_watch.py:watch():39] Watching
